Crashlytics/Crashlytics/Controllers/FIRCLSExistingReportManager.m

```diff
@@ -52,15 +58,15 @@ - (instancetype)initWithManagerData:(FIRCLSManagerData *)managerData
   return self;
 }
 
-NSInteger compareOlder(FIRCLSInternalReport *reportA,
+NSInteger compareNewer(FIRCLSInternalReport *reportA,
                        FIRCLSInternalReport *reportB,
                        void *context) {
-  return [reportA.dateCreated compare:reportB.dateCreated];
+  return -1 * [reportA.dateCreated compare:reportB.dateCreated];
```

Comment:
Should there be a comment here noting that you are inverting the standard comparison result, or possibly swap A and B instead of multiplying by `-1`?

Reply:
Ooh good call